code/src/MyThought.js

```diff
@@ -0,0 +1,50 @@
+import React, { useState } from 'react'
+import { Emoji } from './Emoji'
+
+export const MyThought = ({ setThoughts }) => {
+  const [myThought, setMyThought] = useState('')
+
+  const handleFormSubmit = (event) => {
+    event.preventDefault()
+    fetch('https://technigo-thoughts.herokuapp.com/', {
+      method: 'POST',
+      headers: { 'Content-Type': 'application/json' },
+      body: JSON.stringify({ message: myThought })
+    })
+      .then((res) => res.json())
+      .then((newThought) => {
+        setThoughts((previousThoughts) => [newThought, ...previousThoughts])
+        setMyThought('') // will empty the textarea when the form is submitted
+      })
+  }
+
+  const countCharacters = () => {
+    const textEntered = document.getElementById('myThought').value;
+    const counter = (140 - (textEntered.length));
+    const countRemaining = document.getElementById('countdown')
```

Comment:
As you are ending the other lines with `;`, it would be nice if this follow the same structure
```suggestion
    const countRemaining = document.getElementById('countdown');
```